internal/config/nsmgr/nsmgr.go

```diff
@@ -231,6 +225,12 @@ func (mgr *NamespaceManager) NamespaceFromProcEntry(pid int, nsType NSType) (_ N
 	return GetNamespace(pinnedNamespace, nsType)
 }
 
+// dirForType returns the sub-directory for that particular NSType
+// which is of the form `$namespaceDir/$nsType+"ns"`
+func (mgr *NamespaceManager) dirForType(ns NSType) string {
+	return filepath.Join(mgr.namespacesDir, string(ns)+"ns")
+}
+
```

Comment:
Ah! This should be part of the commit where you introduce `NamespaceFromProcEntry`.